api/datastore/sql/sql.go

```diff
@@ -445,15 +511,17 @@ func (ds *SQLStore) GetAppByID(ctx context.Context, appID string) (*models.App,
 
 // GetApps retrieves an array of apps according to a specific filter.
 func (ds *SQLStore) GetApps(ctx context.Context, filter *models.AppFilter) ([]*models.App, error) {
-	res := []*models.App{}
+	var res []*models.App
```

Comment:
for json we were at least in v1 returning an empty list. a nil list will be nil in json. we should probably have a package level var to skip the alloc, but that's the rationale if we're keeping that behavior

Reply:
ah good point !
I just auto-refactoerd there. 

We should check in DS tests here.